tests/e2e/mariadb/01-connect.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: batch/v1
```

Comment:
I like that we test with a connect pod, but I think for future tests we could also check for specifics like:
* are there quota annotations on the namespace
* does the resizing work and adjust the pvcs
* is the claim SYNCED/HEALTHY -> I would actually add this assert now already, because that one will catch a lot of stuff right out of the bat

Reply:
I'm still in the middle of working for the e2e tests. I just commited what I aleady had so I don't loose work.
The whole e2e stuff will be worked on today.